ykllvmwrap/src/ykllvmwrap.cc

```diff
@@ -236,9 +238,18 @@ extern "C" void *__ykllvmwrap_irtrace_compile(char *FuncNames[], size_t BBs[],
     auto It = F->begin();
     std::advance(It, BBs[Idx]);
     BasicBlock *BB = &*It;
+
     // Iterate over all instructions within this block and copy them over
     // to our new module.
     for (auto I = BB->begin(); I != BB->end(); I++) {
+      // If we've returned from a call skip ahead to the instruction where we
+      // left off.
```

Comment:
If we have multiple calls to the same function in the same basic block, does this handle them correctly?

Reply:
It should, but I will add another test to make sure.

Reply:
Added another test in 3a513a3.